packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -18,7 +18,8 @@ import * as _ from 'lodash';
 import * as PropTypes from 'prop-types';
 import * as React from 'react';
 
-import { lockBodyScroll, unlockBodyScroll } from './utils';
```

Comment:
Are these removed utils still used in northstar ?

Reply:
Nops